ssv/pre_consensus_justification.go

```diff
@@ -1,35 +1,23 @@
 package ssv
 
 import (
-	"github.com/attestantio/go-eth2-client/spec/phase0"
 	"github.com/bloxapp/ssv-spec/qbft"
 	"github.com/bloxapp/ssv-spec/types"
 	"github.com/pkg/errors"
 )
 
-// correctQBFTState returns true if QBFT controller state requires pre-consensus justification
-func (b *BaseRunner) correctQBFTState(msg *qbft.SignedMessage) bool {
-	inst := b.QBFTController.InstanceForHeight(b.QBFTController.Height)
-	decidedInstance := inst != nil && inst.State != nil && inst.State.Decided
-
-	// firstHeightNotDecided is true if height == 0 (special case) and did not start yet
-	firstHeightNotDecided := inst == nil && b.QBFTController.Height == msg.Message.Height && msg.Message.Height == qbft.FirstHeight
-
-	// notFirstHeightDecided returns true if height != 0, height decided and the message is for next height
-	notFirstHeightDecided := decidedInstance && msg.Message.Height > qbft.FirstHeight && b.QBFTController.Height+1 == msg.Message.Height
-
-	return firstHeightNotDecided || notFirstHeightDecided
-}
-
-// shouldProcessingJustificationsForHeight returns true if pre-consensus justification should be processed, false otherwise
-func (b *BaseRunner) shouldProcessingJustificationsForHeight(msg *qbft.SignedMessage) bool {
```

Comment:
why  did we remove all those checks? not all message carry pre-consensus justification

Reply:
We now have simplified this. Compare more closely with `ShouldProcessJustifications`